File: smart-status-player-with-ci-debug/smart-status-player-with-ci/app/src/main/java/com/playface/v2/MainActivity.java
package com.playface.v2;

import android.net.Uri;
import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.MediaController;
import android.widget.Toast;
import android.widget.VideoView;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private VideoView videoView;
    private EditText edtUrl;
    private Button btnPlay;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        videoView = findViewById(R.id.video_view);
        edtUrl = findViewById(R.id.edt_url);
        btnPlay = findViewById(R.id.btn_play);

        MediaController mc = new MediaController(this);
        mc.setAnchorView(videoView);
        videoView.setMediaController(mc);

        btnPlay.setOnClickListener(v -> {
            String url = edtUrl.getText().toString().trim();
            if (url.isEmpty()) {
                Toast.makeText(this, "Masukkan URL video", Toast.LENGTH_SHORT).show();
                return;
            }
            try {
                videoView.setVideoURI(Uri.parse(url));
                videoView.requestFocus();
                videoView.start();
            } catch (Exception e) {
                Toast.makeText(this, "Gagal play: " + e.getMessage(), Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: smart-status-player-with-ci-debug/smart-status-player-with-ci/app/src/main/java/com/example/smartstatus/MainActivity.kt
package com.example.smartstatus

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.provider.Settings
import androidx.appcompat.app.AppCompatActivity
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val btnOverlay = Button(this).apply { text = "Enable Overlay Permission" }
        val btnAccessibility = Button(this).apply { text = "Enable Accessibility" }

        val layout = android.widget.LinearLayout(this).apply {
            orientation = android.widget.LinearLayout.VERTICAL
            addView(btnOverlay)
            addView(btnAccessibility)
        }
        setContentView(layout)

        btnOverlay.setOnClickListener {
            if (!Settings.canDrawOverlays(this)) {
                val intent = Intent(
                    Settings.ACTION_MANAGE_OVERLAY_PERMISSION,
                    Uri.parse("package:$packageName")
                )
                startActivity(intent)
            } else Toast.makeText(this, "Overlay sudah diizinkan", Toast.LENGTH_SHORT).show()
        }

        btnAccessibility.setOnClickListener {
            startActivity(Intent(Settings.ACTION_ACCESSIBILITY_SETTINGS))
            Toast.makeText(this, "Aktifkan SmartStatusPlayer di Accessibility", Toast.LENGTH_LONG).show()
        }
    }
}
